modules/tinymce/src/themes/silver/test/ts/browser/editor/toolbar/ToolbarDrawerToggleTest.ts

```diff
@@ -37,30 +38,33 @@ describe('browser.tinymce.themes.silver.editor.toolbar.ToolbarDrawerToggleTest',
   };
 
   context(`Using the 'ToggleToolbarDrawer' command should toggle the toolbar if applicable`, () => {
-    Arr.each<{ mode: ToolbarMode; shouldToggle: boolean }>([
-      { mode: 'floating', shouldToggle: true },
-      { mode: 'sliding', shouldToggle: true },
-      { mode: 'wrap', shouldToggle: false },
-      { mode: 'scrolling', shouldToggle: false }
-    ], (test) => {
+    const isWindowsAndEdge = Env.os.isWindows() && Env.browser.isChromiumEdge();
```

Comment:
What's going on here? Why do we suddenly need to know the difference between Chrome and Edge?

Reply:
We discussed a weird flake that is only happening on Edge. So trying to skip it for now it's unrelated to the test. Not sure what to do with these flakes it takes lots of time to investigate them. I would probably not be something a user will run into.